.github/workflows/deploy-image.yaml

```diff
@@ -0,0 +1,106 @@
+name: deploy-image
+on:
+  workflow_dispatch:
+#    inputs:
+#      distro:
+#        description: Base image distro
+#        required: true
+#        default: debian
+#      imageVersion:
+#        description: Base distro image version/release
+#        required: true
+#        default: bullseye
+#      imageVariant:
+#        description: Base image variant
+#        required: true
+#        default: slim
+  workflow_run:
+    workflows:
+      - build-latest
+    branches:
+      - develop
+    types:
+      - completed
+jobs:
+  deploy-image:
+    runs-on: ubuntu-latest
+    env:
+      latest-ref: refs/heads/develop
+    strategy:
+      matrix:
+        postgresMajorVersion:
+          - 13
+        postgisMajorVersion:
+          - 3
+        postgisMinorRelease:
+          - 1
+        include:
+          - distro: debian
+            imageVersion: bullseye
+            imageVariant: slim
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Get build cache
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          # Build inputs are:
+          # - Dockerfile recipe
+          # - docker-compose.build.yml
+          # - build args (.example.env)
+          # - base_build directory
+          # - scripts directory
+          key: buildx-${{ hashFiles('Dockerfile', 'docker-compose.build.yml', '.example.env', 'base_build', 'scripts') }}-${{ github.sha }}
+          restore-keys: |
+            buildx-${{ hashFiles('Dockerfile', 'docker-compose.build.yml', '.example.env', 'base_build', 'scripts') }}-
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build base image
+        id: docker_build_base
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKERHUB_REPO }}/postgis:base-${{ matrix.distro }}-${{ matrix.imageVersion }}-${{ matrix.imageVariant }}
```

Comment:
RFC to @NyakudyaA .

I propose we switch the image tag from `debian-bullseye-slim-base` into `base-debian-bullseye-slim`, due to the public convention that image variant is always the last, and `-` separates context from generic to specific (`base` being more generic usage).

Ideally there should be also the latest base tag which only spells `base`, like public convention.